core/src/main/java/org/elasticsearch/gateway/MetaDataStateFormat.java

```diff
@@ -270,6 +267,8 @@ public  T loadLatestState(ESLogger logger, Path... dataLocations) throws IOExcep
                             files.add(pav);
                         }
                     }
+                } catch (NoSuchFileException | FileNotFoundException ex) {
+                    // no _state directory -- move on
```

Comment:
@nik9000 yes we check multiple paths it only has to be on one. The code is equivalent in that regard we did a continue if the dir wasn't here....